### PR TITLE
Create 'scale' operation when scaling container count

### LINKED ISSUE #141: Create an operation after changing a service's containerCount
refs #129

### PR DESCRIPTION
**Pull request by bantic**
Fixes #141

@mixonic for you

### CODE REVIEW

File: tests/.jshintrc [hunk 1]
```diff
@@ -62,8 +62,6 @@
     "clickPrevPageLink",
     "locationUpdatedTo",
     "titleUpdatedTo",
-    "setNoUISlider",
-    "slideNoUISlider",
```
Comment (bantic) [inline tests/.jshintrc]: there actually already was a `triggerSlider` helper :P

File: tests/acceptance/apps/show-test.js [hunk 2]
```diff
@@ -90,100 +90,3 @@ test('visiting /apps/my-app-id when the app is deprovisioned', function() {
...
-      handle: 'hubot-service',
-      container_count: 5
-    });
-  });
-
-  signInAndVisit('/apps/' + appId + '/services');
-  slideNoUISlider('.slider', 5);
-  setNoUISlider('.slider', 5);
-  click('button:contains(Scale)');
-});
```
Comment (bantic) [inline tests/acceptance/apps/show-test.js]: these tests already existed in (or were just added to) the apps-services-tests.

File: tests/helpers/aptible-helpers.js [hunk 3]
```diff
@@ -449,5 +435,7 @@ Ember.Test.registerHelper('stubUser', function(app, userData={}){
 });
 
 Ember.Test.registerAsyncHelper('triggerSlider', function(app, selector, argument){
-  Ember.$(selector).trigger('slide', argument);
+  let slider = findWithAssert(selector);
+  slider.trigger('slide', argument);
+  slider.trigger('set', argument);
```
Comment (bantic) [inline tests/helpers/aptible-helpers.js]: appears to be no harm in trigger set & slide together, and the 'set' is necessary for the `service-scaler` component to show its 'scale' button

File: tests/.jshintrc (hunk 1 above)
Comment (mixonic) [inline tests/.jshintrc]: *sigh*

**Pull request merged**